src/lib/__tests__/VirtualScroller-test.js

```diff
@@ -464,14 +463,15 @@ describe('VirtualScroller', () => {
 
         beforeEach(() => {
             stubs.dispatchEvent = jest.fn();
-            scrollingEl = { remove: () => {}, dispatchEvent: stubs.dispatchEvent };
+            scrollingEl = { dispatchEvent: stubs.dispatchEvent };
 
             virtualScroller.totalItems = 10;
             virtualScroller.itemHeight = 10;
             virtualScroller.margin = 0;
             virtualScroller.scrollingEl = scrollingEl;
 
             stubs.isVisible = jest.spyOn(virtualScroller, 'isVisible').mockImplementation();
+            stubs.removeChild = jest.spyOn(virtualScroller.anchorEl, 'removeChild').mockImplementation();
```

Comment:
Is this needed? Not sure I see it being used in any test

Reply:
No, not really. I did it so it would fit in better with the other spy on the line above. I can remove it if it's concerning.

Reply:
Not concerning, already approved